src/category_theory/limits/shapes/pullbacks.lean

```diff
@@ -213,6 +213,18 @@ begin
   erw [t.w inl, ← t.w inr], refl
 end
 
+/-- To check whether a morphism is equalized by the maps of a pullback cone, it suffices to check
+  it for `fst t` and `snd t` -/
+lemma ext (t : pullback_cone f g) {W : C} {k l : W ⟶ t.X}
```

Comment:
I'm not really sure if this should be called `ext`. @semorrison do you have some sort of convention for this?

Reply:
I agree this shouldn't be called `ext`, but I'm stumped as to a good name right now...

Reply:
Maybe just `equalized_ext`. I don't know.

Reply:
```suggestion
lemma equalizer_ext (t : pullback_cone f g) {W : C} {k l : W ⟶ t.X}
```